### PR TITLE
Doc update - remove media-breakpoint-up(xs)

### PR DESCRIPTION
**Pull request by DavidDeSloovere**
If media-breakpoint-down(xl) is not listed, then media-breakpoint-up(xs) does not need to be listed either.
The above 4 media queries are now aligned with the 4 media-breakpoint-up mixins (just like their media-breakpoint-down counterparts).

Also added example for media-breakpoint-down.

This PR is a result of questions from a colleague that misinterpreted the existing docs. This should clarify it.

### CODE REVIEW

File: docs/4.1/layout/overview.md [hunk 1]
```diff
@@ -70,13 +70,12 @@ Bootstrap primarily uses the following media query ranges—or breakpoints—in
 Since we write our source CSS in Sass, all our media queries are available via Sass mixins:
 
 {% highlight scss %}
-@include media-breakpoint-up(xs) { ... }
```
Comment (mdo) [inline docs/4.1/layout/overview.md]: Let's actually swap this with something that explicitly states there's no `xs` query?

```scss
// No media query necessary for `xs` breakpoint as it's effectively `@media (min-width: 0) { ... }
```

File: docs/4.1/layout/overview.md [hunk 2]
```diff
@@ -70,13 +70,12 @@ Bootstrap primarily uses the following media query ranges—or breakpoints—in
...
 @include media-breakpoint-up(sm) { ... }
 @include media-breakpoint-up(md) { ... }
 @include media-breakpoint-up(lg) { ... }
 @include media-breakpoint-up(xl) { ... }
```
Comment (mdo) [inline docs/4.1/layout/overview.md]: Let's insert an example then to back up that `xs` comment I suggested earlier.

File: docs/4.1/layout/overview.md (hunk 1 above)
Comment (DavidDeSloovere) [reply docs/4.1/layout/overview.md]: Good point. People might think 'where is xs'.

File: docs/4.1/layout/overview.md (hunk 2 above)
Comment (DavidDeSloovere) [reply docs/4.1/layout/overview.md]: Added example, hope I interpreted the comment correctly.

**Comment by DavidDeSloovere** [comment]
I don't know why the build is failing for just a docs update.

**Comment by wolfy1339** [comment]
Maybe you should check. Because even docs have linting and builds too 😃 
Anyways, here's the error `docs/4.1/layout/overview/index.html: line 463, col 1, indenting spaces must be used in groups of 2`

**Comment by DavidDeSloovere** [comment]
🤦‍♂️  Well at least I learned something.
I thought was related to the branch being out of date and not being able to update. Was more other way around.

**Pull request merged**